src/azure/storage/signer.rs

```diff
@@ -67,6 +67,14 @@ impl Signer {
                 ctx.query_append(token);
                 return Ok(ctx);
             }
+            Credential::BearerToken(token) => {
+                ctx.headers.insert(AUTHORIZATION, {
```

Comment:
We can't set header directly without check SigningMethod since we could leak this to user unexpectedly.

Reply:
🧑🏾‍💻 update it.